src/assets/regions.json

```diff
@@ -0,0 +1,64 @@
+{
+    "regions": [
+        "Akmenės rajono savivaldybė",
+        "Alytaus miesto savivaldybė",
+        "Alytaus rajono savivaldybė",
+        "Anykščių rajono savivaldybė",
+        "Birštono savivaldybė",
+        "Biržų rajono savivaldybė",
+        "Druskininkų savivaldybė",
+        "Elektrėnų savivaldybė",
+        "Ignalinos rajono savivaldybė",
+        "Jonavos rajono savivaldybė",
+        "Joniškio rajono savivaldybė",
+        "Jurbarko rajono savivaldybė",
+        "Kaišiadorių rajono savivaldybė",
+        "Kalvarijos savivaldybė",
+        "Kauno miesto savivaldybė",
+        "Kauno rajono savivaldybė",
+        "Kazlų Rūdos savivaldybė",
+        "Kėdainių rajono savivaldybė",
+        "Kelmės rajono savivaldybė",
+        "Klaipėdos miesto savivaldybė",
+        "Klaipėdos rajono savivaldybė",
+        "Kretingos rajono savivaldybė",
+        "Kupiškio rajono savivaldybė",
+        "Lazdijų rajono savivaldybė",
+        "Marijampolės savivaldybė",
+        "Mažeikių rajono savivaldybė",
+        "Molėtų rajono savivaldybė",
+        "Neringos savivaldybė",
+        "Pagėgių savivaldybė",
+        "Pakruojo rajono savivaldybė",
+        "Palangos miesto savivaldybė",
+        "Panevėžio miesto savivaldybė",
+        "Panevėžio rajono savivaldybė",
+        "Pasvalio rajono savivaldybė",
+        "Plungės rajono savivaldybė",
+        "Prienų rajono savivaldybė",
+        "Radviliškio rajono savivaldybė",
+        "Raseinių rajono savivaldybė",
+        "Rietavo savivaldybė",
+        "Rokiškio rajono savivaldybė",
+        "Skuodo rajono savivaldybė",
+        "Šakių rajono savivaldybė",
+        "Šalčininkų rajono savivaldybė",
+        "Šiaulių miesto savivaldybė",
+        "Šiaulių rajono savivaldybė",
+        "Šilalės rajono savivaldybė",
+        "Šilutės rajono savivaldybė",
+        "Širvintų rajono savivaldybė",
+        "Švenčionių rajono savivaldybė",
+        "Tauragės rajono savivaldybė",
+        "Telšių rajono savivaldybė",
+        "Trakų rajono savivaldybė",
+        "Ukmergės rajono savivaldybė",
+        "Utenos rajono savivaldybė",
+        "Varėnos rajono savivaldybė",
+        "Vilkaviškio rajono savivaldybė",
+        "Vilniaus miesto savivaldybė",
+        "Vilniaus rajono savivaldybė",
+        "Visagino savivaldybė",
+        "Zarasų rajono savivaldybė"
+    ]
```

Comment:
are you intending to keep this all here or is it just temporary?

Reply:
keeping probs